### ISSUE TITLE
Access to binding from observable

### ISSUE THREAD

**Issue by Gle2nn** [knockout/knockout#2276]
Hi. I am working on a custom binding for AutoNumeric 4.0.0. This version of AutoNumeric creates a JS object that is not attached to the DOM. I am using the 'createBindingHandler' pattern (see #1999) to keep track of this object.
I am encountering a few cases wherein I would like to change the AutoNumeric options for various input fields on my forms based on the value in another input. Previously, I could find the relevant AutoNumeric objects via the DOM using something like: $( "#fieldID" ).autoNumeric(), but now I am hoping to access the objects via the observable bindings.
Is this behaviour supported by KO? Is there another way to accomplish this (for example, maybe attaching the AutoNumeric object to the DOM is the best way)?
Thanks!

**Comment by mbest** [comment]
There's no reason why your custom bindings can't continue to use jQuery to interact with the DOM. You can definitely have related bindings that read data from the DOM.

Knockout allows a lot of flexibility in how your coding logic is divided between the view, bindings, and view model.

**Comment by Gle2nn** [comment]
Thanks for the reply.
I certainly understand that I can use jQuery to work with the DOM to 
accomplish my goal. In fact I implemented my binding using that 
technique earlier today and it works fine.
However, it feels like using the DOM to get the data I need is "taking 
the long way around" so to speak.
I was hoping to be able to use something like: 
myView.myObservable.getBinding().updateConfiguration( { <my new options 
here> } );
It just feels more natural than finding the right DOM object, then the 
related data, then doing the reconfiguration.
Thanks again,
Gle2nn

**Comment by Gle2nn** [comment]
FYI I dug into this a bit deeper today and came up with the following implementation:

I added the following code to the `init` routine of the `createBindingHandler` after the creation of the binding object (newObj):
```javascript
if( ko.isObservable( valueAccessor() ) )
{
	valueAccessor().getBinding = function(){ return( newObj ); };
}
```
I added the following code to the binding prototype object (obj):
```javascript
getAutoNumeric: function()
{
	return( this.anElement );	//The AutoNumeric object created for this instance of the binding.
}
```
With these simple additions, I can write the following in my view model:
```javascript
viewModel.numericField.getBinding().getAutoNumeric().update( { decimalPlaces: 3 } );
```
With this code I avoid

- stuffing the AutoNumeric object into the DOM to avoid losing the reference,
- avoid searching the DOM for the elements containing the AutoNumeric object references when I need to reconfigure them
- avoid a simple dependency on JQuery.

Maybe this will help someone else facing a similar problem. Comments?
